components/CardMenu/CardMenu.styled.js

```diff
@@ -0,0 +1,25 @@
+import styled from "styled-components";
+import { MenuButton, MenuItem } from "@szhsin/react-menu";
+
+export const StyledMenuButton = styled(MenuButton)`
+  border: none;
+  background-color: transparent;
+  margin-left: 80%;
+`;
+
+export const IconWrapper = styled.div`
+  font-size: 2.3rem;
+  color: var(--color-font-3);
+  opacity: 0.9;
+  padding: 1rem 1rem 0 1rem;
+`;
+
+export const StyledMenu = styled.div`
+  padding: 1.5rem;
+  margin-right: 3rem;
+`;
+
+export const StyledMenuItem = styled(MenuItem)`
+  padding: 1rem 1rem 1rem 0.5rem;
```

Comment:
if padding left and right are the same, you can shorten this to `padding: 1rem 1rem 0.5rem`